### PR TITLE
Mikekry/6136 not mandatory localizationset

### PR DESCRIPTION
**Pull request by MikeKry**
issue #6136 

Allows to filter content items by culture in GraphQL

Use cases: 
listing of content items in given language

Without this, you need to first get localizationSet and then filter individually by each localizationSet.

### CODE REVIEW

**Comment by MikeKry** [comment]
Is there anything else needed before merge?

Just asking because it is my first PR here, so I do not know, how long it takes before merge and release of new nugget packages. Would be nice to have this merged for my project.

File: src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public LocalizationInputObjectType(IStringLocalizer<LocalizationInputObjectType>
             Description = S["the localization part of the content item"];
 
             AddScalarFilterFields<StringGraphType>("culture", S["the culture of the content item to filter"]);
-            Field(x => x.LocalizationSet).Description(S["the localization set of the content item to filter"]);
+            Field(x => x.LocalizationSet, true).Description(S["the localization set of the content item to filter"]);
```
Comment (carlwoodhouse) [inline src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs:16]: can you use a named param? eg `nullable: true` - it helps someone reading the code to understand what the overload is doing,

Comment (carlwoodhouse) [inline src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs:16]: can you use a named param? eg `nullable: true` - it helps someone reading the code to understand what the overload is doing and protect us against overload changes between graphql versions,

**Review by carlwoodhouse — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs (hunk 1 above)
Comment (MikeKry) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs]: @carlwoodhouse 

Sure, no problem, done.

Comment (carlwoodhouse) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs]: cheers

**Review by carlwoodhouse — APPROVED** [review]

**Pull request merged**